chef/site-cookbooks/wca/libraries/wca_helper.rb

```diff
@@ -12,11 +12,14 @@ def self.get_username_and_repo_root(recipe)
   end
 
   def self.get_secrets(recipe)
-    if recipe.node.chef_environment.start_with?("development")
+    if recipe.node.chef_environment == "development"
```

Comment:
We used to have multiple kinds of development environments, but that is long gone, so I took this opportunity to simplify things.